### ISSUE TITLE
Format rent and income "details" text in Airtable

### ISSUE THREAD

**Issue by trevorshannon** [philhawksworth/the-united-effort-orginization#45]
The current max income details field in Airtable is a text field with text formatting that is not well-suited to the new website's display method.  We should change this text formatting to make it how we want and also consistent across unit listings.  

The same goes for the rent details.

**Comment by trevorshannon** [comment]
Updated newly imported development Airtable (pending promotion to production) to allow for better formatting of this detail text.

See an example of the new hover tooltip format here:
https://cleanup-db-updates--ueo.netlify.app/housing/affordable-housing/1041

**Comment by trevorshannon** [comment]
In order to do this, I deprecated `max_income_details` and made a few new fields.  There are 12 max income fields, one for each household size 1-12 people.  I populated those fields with the values that were manually typed into the old `max_income_details` field. 

I also made the fields `MAX_YEARLY_INCOME_LOW_USD` and `MAX_YEARLY_INCOME_HIGH_USD` that calculate the low and high end of the max income range.  Those values are displayed in the rent tables on the website.

If max income by household size is not known, it is also possible to override the low and high max incomes.  I made `OVERRIDE_MAX_YEARLY_INCOME_LOW_USD` and `OVERRIDE_MAX_YEARLY_INCOME_HIGH_USD` for that purpose.

Finally, there were a few properties with max income notes that did not directly map to actual max income numerical values.  So I made `MAX_INCOME_NOTES`
